client/components/select-dropdown/item.jsx

```diff
@@ -15,6 +15,7 @@ class SelectDropdownItem extends Component {
 		disabled: PropTypes.bool,
 		icon: PropTypes.element,
 		ariaLabel: PropTypes.string,
+		extra: PropTypes.element,
```

Comment:
I wonder.

Since this is fundamentally an icon, should we call this prop something like `secondaryIcon` and embrace the fact that... it's an icon! 😄

Reply:
My idea was to not have this limited to an icon, since the dropdown is a shared component I didn't want to tie it to a specific naming.

Reply:
Changed the naming since I was changing some other things.